site/src/docs/components/hero/tabs.mdx

```diff
@@ -0,0 +1,33 @@
+---
+slug: '/components/hero/tabs'
+title: 'Hero'
+---
+
+import { StatusLabel } from 'hds-react';
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Layout from '../../../components/layout';
+import PageTabs from '../../../components/PageTabs';
+
+# Hero
+
+<StatusLabel type="info">Stable</StatusLabel>
```

Comment:
Not stable but Beta? One of these anyway:
> Draft – The component is work in progress
Beta – The component is being tested

Reply:
I changed it to Draft, because we haven't used Beta with other components yet. Maybe we need to take a closer look of all of our components and then change the statuses according to the new parameters with 3.0.0.